package/MDAnalysis/coordinates/memory.py

```diff
@@ -326,8 +326,12 @@ def __init__(self, coordinate_array, order='fac',
             self.coordinate_array.shape[self.stored_order.find('a')]
 
         if velocities is not None:
-            if velocities.ndim == 2:
-                velocities = velocities[np.newaxis, :, :]
+            try:
+                if velocities.ndim == 2:
+                    velocities = velocities[np.newaxis, :, :]
+            except AttributeError:
```

Comment:
This just checks whether the `velocities` object has an attribute `ndim`. For type checking, better use `isinstance()`. But since that's not really "pythonic" anyway, I'd prefer calling `numpy.asarray(velocities)` before doing attribute checks.